.gitignore

```diff
@@ -4,3 +4,5 @@ bash.exe.stackdump
 .vs
 *.orig
 *.param
+.scannerwork/
+.sonarqube/
```

Comment:
These are temporary folders created by `sonnar-scanner` that we don't want in source control